resources/states.raml

```diff
@@ -87,3 +87,54 @@ post:
         body:
           application/json:
             example: !include ../examples/state.example.json
+/key={key}:
+  (methodName): withKey
+  type:
+    baseResource:
+      uriParameterName: Key
+      resourceType: State
+      resourceUpdateType: Update
+  get:
+    securedBy:
+      [
+        oauth_2_0:
+          {
+            scopes: ['manage_project:{projectKey}', 'view_orders:{projectKey}'],
```

Comment:
probably you should add "view_states:{projectKey}", because "view_orders:{projectKey}" is the deprecated one. See https://docs.commercetools.com/http-api-projects-states#get-state-by-key